package.json

```diff
@@ -18,7 +18,7 @@
     "test:windows": "ember try:one %EMBER_TRY_SCENARIO% test --skip-cleanup",
     "test:node:dev": "BUILD_DEV=true testem -f testem-node.js",
     "test:null": "echo 'no appropriate changes detected, not running tests'",
-    "prepublish": "node ./bin/install-test-addons.js"
+    "postinstall": "node ./bin/install-test-addons.js"
```

Comment:
I'm unsure about this. `postinstall` runs for all consumers also (we picked `prepublish` originally since it only runs when you `npm install` in this repo, not `npm install ember-engines`).

Maybe with your other changes, this isn't needed?

Reply:
Ah, I knew there was a reason we had used prepublish originally. I just couldn't remember what it was.

I think you're right though, that this shouldn't be needed after the other changes. Will remove. Hopefully, it'll be ready to land after that.